apisix/cli/ops.lua

```diff
@@ -560,6 +560,7 @@ Please modify "admin_key" in conf/config.yaml .
     end
     sys_conf["wasm"] = yaml_conf.wasm
 
+    sys_conf["config_center"] = yaml_conf.apisix.config_center
```

Comment:
Handled in https://github.com/apache/apisix/blob/15517feb55a0c5e87c5fdb3b91d647ee0ca1c96f/apisix/cli/ops.lua#L555

Reply:
rm this